routes/usersRoutes.js

```diff
@@ -0,0 +1,16 @@
+import express from 'express';
+
+import { authenticateUserToken } from '../middlewares/authorization.js';
+import userControllers from '../controllers/userControllers.js';
+import userAuthControllers from '../controllers/userAuthControllers.js';
+
+const router = express.Router();
+
```

Comment:
You can create 2 different route files to separate them, and then an index.js to export it to the whole app. This will improve the readability.